### PR TITLE
Bug 1202442 - Update l20n.js to 3.0.6. r=gandalf

### PR DESCRIPTION
**Pull request by stasm**
https://bugzilla.mozilla.org/show_bug.cgi?id=1202442

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ecb9e3d5b01808303a3a3e80f3b72015ad0941fb

File: apps/fm/js/fm.js [hunk 1]
```diff
@@ -885,9 +885,8 @@ function init() {
 
 document.addEventListener('DOMLocalized', function() {
   // localize MHz
-  document.l10n.format('mhz').then(str => {
-    mhzString = str.value;
-    $('frequency').dataset.unit = mhzString;
+  document.l10n.formatValue('mhz').then(str => {
+    $('frequency').dataset.unit = mhzString = str;
```
Comment (hfiguiere) [inline apps/fm/js/fm.js]: I see no point in changing anything else than line 888 and 889. At least now it is consistent with the documentation.
You are just making the code less readable.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6f8fe0eb19b799904a88de94260ef33abdf55b56

File: apps/fm/js/fm.js (hunk 1 above)
Comment (stasm) [inline apps/fm/js/fm.js]: I brought back the explicit assignment to `mhzString` in the updated version of the pr.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ae6c57041dfe0d36afb0b690fd0f09368b219841

**Pull request merged**